src/main/java/com/unzer/payment/Customer.java

```diff
@@ -155,15 +155,21 @@ public PaymentType map(PaymentType paymentType, JsonObject jsonObject) {
 		return null;
 	}
 
-	public Address getShippingAddress() {
+	public ShippingAddress getShippingAddress() {
 		return shippingAddress;
 	}
 
-	public Customer setShippingAddress(Address shippingAddress) {
+	public Customer setShippingAddress(ShippingAddress shippingAddress) {
 		this.shippingAddress = shippingAddress;
 		return this;
 	}
 
+	@Deprecated
```

Comment:
I am not even sure if we need to deprecated the old setter. the addtional the shipping data are not always needed. Maybe we discuss that separately.